lib/rucio/core/replica.py

```diff
@@ -35,8 +35,8 @@
 # - Eli Chadwick <eli.chadwick@stfc.ac.uk>, 2020
 # - Patrick Austin <patrick.austin@stfc.ac.uk>, 2020
 # - Eric Vaandering <ewv@fnal.gov>, 2020-2021
-# - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020
-# - Radu Carpa <radu.carpa@cern.ch>, 2021
+# - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020-2021
+# - rcarpa <2905943+rcarpa@users.noreply.github.com>, 2021
```

Comment:
?

Reply:
git author… I will just revert this file, I did no real change here.